src/deprecation.h

```diff
@@ -10,7 +10,7 @@
 // Per https://zips.z.cash/zip-0200
 // Shut down nodes running this version of code, 16 weeks' worth of blocks after the estimated
 // release block height. A warning is shown during the 14 days' worth of blocks prior to shut down.
-static const int APPROX_RELEASE_HEIGHT = 1504042;
+static const int APPROX_RELEASE_HEIGHT = 1540976;
```

Comment:
Hmm, I would have left this change as-is, because it's the approx release height of 4.6.0-2, not 4.7.0-rc1. But it doesn't really matter; either of them will be earlier than the eventual 4.7.0 release.

Reply:
My reasoning was that the release height should never go backwards between releases, according to the time they are released. So the release height will need to be at least 1540976 whatever happens, and we haven't decided to make it later than that. It's not critical though.